### PR TITLE
Task/script capability http request handler

### PR DESCRIPTION
**Pull request by arthware**
#### Check List:
* Unit tests: YES 
* Documentation: NO

### CODE REVIEW

**Comment by arthware** [comment]
Note: Builds on top of PR #3392. #3396 should be merged first.

File: modules/flowable-bpmn-model/src/main/java/org/flowable/bpmn/model/AbstractFlowableHttpHandler.java [hunk 1]
```diff
@@ -14,17 +14,20 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
```
Comment (dbmalkovsky) [inline modules/flowable-bpmn-model/src/main/java/org/flowable/bpmn/model/AbstractFlowableHttpHandler.java:17]: I don't see `Objects` used in this class.

File: modules/flowable-engine/src/main/java/org/flowable/engine/impl/bpmn/http/handler/ScriptHttpHandler.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
+package org.flowable.engine.impl.bpmn.http.handler;
```
Comment (dbmalkovsky) [inline modules/flowable-engine/src/main/java/org/flowable/engine/impl/bpmn/http/handler/ScriptHttpHandler.java:1]: Missing license header

File: modules/flowable-bpmn-model/src/main/java/org/flowable/bpmn/model/AbstractFlowableHttpHandler.java (hunk 1 above)
Comment (arthware) [reply modules/flowable-bpmn-model/src/main/java/org/flowable/bpmn/model/AbstractFlowableHttpHandler.java]: fixed

File: modules/flowable-bpmn-converter/src/main/java/org/flowable/bpmn/converter/child/ScriptInfoParser.java [hunk 3]
```diff
@@ -20,6 +20,7 @@
 import org.flowable.bpmn.model.BaseElement;
 import org.flowable.bpmn.model.BpmnModel;
 import org.flowable.bpmn.model.FlowableListener;
+import org.flowable.bpmn.model.HasScriptInfo;
```
[the remark this answers is not in the document]
Comment (arthware) [reply modules/flowable-bpmn-converter/src/main/java/org/flowable/bpmn/converter/child/ScriptInfoParser.java:22]: fixed

**Pull request merged**